### DISCUSSION TITLE
Inpatient perdiem vs Case rate negotiation

### DISCUSSION THREAD

**Discussion by iGokul-G** [CMSgov/price-transparency-guide#24] [Q&A]
As the in-network layout suggests that the negotiated rate needs to be a dollar amount . There could be payment arrangements ex.  say Hospital ABC  as perdiem ( per day )  for certain items/services vs hospital XYZ  as case rate  ( global ). Do we need any additional field to represent the payment method  or new negotiated type ?

**Comment by shaselton-usds** [comment]
Thanks for the questions!

Can you do me a favor and review this paragraph?: https://www.federalregister.gov/d/2020-24591/p-638 Near the end, a PMPM is discussed. Also, you can search "PMPM" on that page for other references to see if it help clarify anything.

Let me know if that helps.

**Reply by taylorpatriciab** [reply]
@shaselton The question was about inpatient per diem (per day) arrangements.  A PMPM addresses a different situation where reimbursement is based on a Per Member Per Month.  Is there anything that speaks to per diem arrangements?

**Reply by ryan-thomas-peterson** [reply]
@taylorpatriciab , agree with your note.  We have contracts that are based on per diems as well and based on days with a particular qualifying revenue code.  In addition, those per diems may have a different rate after a threshold number of days.  Say $300 / day for days 1-5 at a behavioral health facility and $200 / day after that.  It's not clear how this could be reported.

**Reply by iGokul-G** [reply]
@shaselton-usds .  Do we have  any flexibility  on allowed values  for the tag  "negotiated_type"  to include the  basis of payment  like perdiam  - per day / global   say with in "negotiation_arrangement" as ffs

     "negotiated_price": {
        "negotiated_type": "perdiem ",
        "negotiated_rate": 123.45,
        "expiration_date": "2022-01-01"

Payer's  may have inpatient perdiem payment agreements  like pay $100 for first 3 days then $80 for next 4 days , any thing above $50. Restriction in the layouts may have significant impact in representing such negotiations

**Reply by tdfow** [reply]
Could negotiation_arrangement be expanded to possibly address per diem ("perdiem") or percentage of charge ("percentageofcharge"), where then the negotiated_rate could be published to indicate the perdiem rate or the percentage---since we don't have a claim to indicate the charge amount or the number of days, etc.---then we could publish the 'rate' based on this type of arrangement?

**Reply by ghost** [reply]
@tdfow @shaselton-usds This should be opened up to address other posted questions where the negotiated rate itself is based on a percentage and allowing the reporting of a percentage and not just a dollar amount.

- If negotiated rate is based on percentage of charge--then how would this be reported in the in-network file? #23
- Billing Code Types other than CPT, HCPCS, DRG, ICD, NDC-- how are they reported

? #9

**Reply by dawndrake58** [reply]
We also have per diems that are based on a threshold of days.  Days 1-2 $300, Days 3-5 $250 etc.  In addition, we have multiple per diems for skilled nursing and long term care that pay from a single revenue code, but are differentiated by the ICD-10 Diagnosis or some other variable.  In pulling these it doesn't make sense to enter a single rev code multiple times with multiple rates.  That would make no sense to the end user.

**Reply by jltang-kp** [reply]
@dawndrake58 we do have an exact same scenario for bed days as well. Since the DRGs are exactly the same and the only difference are the bed days. We are planning on using the description field to say Days 1-2, Days 3-5 etc. However an additional field to show the tiered rates may help to further clarify the differences in rate rather than description field.

**Reply by rgouldkc** [reply]
We got per diem and percentage added to negotiated type. IMO we still need to have case rate added as well.
